### ISSUE TITLE
Is it normal I get ads?

### ISSUE THREAD

**Issue by fabeit** [segler-alex/RadioDroid#986]
I get a 30 seconds ad in the language of my IP even though I am connecting to radios from another country and in another language, so definitely IP based ads. 

I thought this app was ad-free? 

I don't get any ads with Transistor for example.

**Comment by werman** [comment]
We don't have any ads. Ads could only be played by stations themselves.

Transistor also gets station's url from the same database as we are. So there should be no difference.

Make sure that you use our build of RadioDroid by following the links from the main page. 

If you still sure you are getting ads where Transistor does not - post the stations you have tested.

**Comment by palopezv** [comment]
FYI, many podcast networks have been using the same business model as radio stations for several years s now. Some embed ads directly while others use specialist services, such as the audio.ad network. I block the latter by adding its several servers to a well-known system-wide FOSS adblocker I use in my phone.

**Comment by autinerd** [comment]
Yeah, these are delivered by the streaming service, there are a few networks who deliver ads within the stream based on your location, and these are not always possible to block. That would happen as well if you play the stream in VLC or other media players.
